### PR TITLE
Issue #140 Convert integration test docs to adoc

### PR DESCRIPTION
**Pull request by jsliacan**
There was only `README.md` to convert. This PR does that with a few minor typo fixes.

### CODE REVIEW

File: test/integration/README.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
+= CodeReady Containers Integration Tests
+:toc: macro
+:toc-title:
+:toclevels:
+
+toc::[]
+
+[[intro-to-integration-testing]]
+== Introduction
+Integration tests use http://github.com/code-ready/clicumber[Clicumber] package which provides basic functionality for testing CLI binaries.
```
Comment (robin-owen) [inline test/integration/README.adoc]:
```adoc
Integration tests use the link:https://github.com/code-ready/clicumber[Clicumber] package to provide basic functionality for testing CLI binaries.
```

File: test/integration/README.adoc [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Clicumber allows to run commands in persistent shell instance (shell, tcsh, zsh, cmd, or powershell), assert its outputs (stdout, stderr or exitcode), check configuration files, etc.
+
+The general functionality of Clicumber is then extended by CRC specific test code to cover the whole functionality of CRC.
```
Comment (robin-owen) [inline test/integration/README.adoc]: We should be using "CodeReady Containers" in documentation, not "CRC". This sentence could also be moved to the end of the previous paragraph, as it's directly related to the contents of that paragraph.

File: test/integration/README.adoc [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+```bash
+$ make integration
+```
+
+
+[[integration-test-logs]]
+== Logs
+
+Test logs can be found in [filename]`test/integration/out/test-results`.
```
Comment (robin-owen) [inline test/integration/README.adoc]: :+1:

File: test/integration/README.adoc [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+:toc-title:
+:toclevels:
+
+toc::[]
+
+[[intro-to-integration-testing]]
+== Introduction
+Integration tests use http://github.com/code-ready/clicumber[Clicumber] package which provides basic functionality for testing CLI binaries.
+
+Clicumber allows to run commands in persistent shell instance (shell, tcsh, zsh, cmd, or powershell), assert its outputs (stdout, stderr or exitcode), check configuration files, etc.
```
Comment (robin-owen) [inline test/integration/README.adoc]: In this list of shell instances, is "shell" meant to refer to Bash?

```adoc
Clicumber allows running commands in a persistent shell instance (`bash`, `tcsh`, `zsh`, Command Prompt, or PowerShell), assert its outputs (standard output, standard error, or exit code), check configuration files, and so on.
```

Comment (jsliacan) [reply test/integration/README.adoc]: You're right, `bash` was meant... Thanks for suggesting alternatives with each comment! It should all be changed now.

**Pull request merged**